specification/cognitiveservices/Language.AnalyzeText-authoring/models.tsp

```diff
@@ -336,16 +322,13 @@ model Error {
   details?: Error[];
 
   /**
-   * An object containing more specific information than the current object about
-   * the error.
+   * An object containing more specific information than the current object about the error.
    */
   innererror?: InnerErrorModel;
 }
 
 /**
- * An object containing more specific information about the error. As per
- * Microsoft One API guidelines -
- * https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.
+ * An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.
```

Comment:
Any reason why you changed the comments to be in single line throughout the PR?

Reply:
The multiple lines here will be several "/n" in the generated swagger, so Heath suggested us to avoid this sort of rendering control characters.